cypress/support/fragments/inventory/inventoryItem/inventoryItems.js

```diff
@@ -0,0 +1,62 @@
+import {
+  Button,
+  SearchField,
+  Pane,
+  Checkbox,
+  Accordion,
+  MultiColumnListCell,
+  MultiColumnList,
+  TextField
+} from '../../../../../interactors';
+
+const searchButton = Button('Search');
+const searchField = SearchField({ id: 'input-inventory-search' });
+const itemStatusAccordion = Accordion({ id:'itemStatus' });
+
+export default {
+  toggleStatus(statusName) {
+    cy.do(itemStatusAccordion.find(TextField({ type:'search' })).fillIn(statusName));
+    // interactor doesn't click
+    cy.get('#itemStatus input').click();
+    cy.do(Checkbox(statusName).click());
+  },
+
+  toggleItemStatusAccordion() {
+    cy.do(itemStatusAccordion.clickHeader());
+  },
+
+  selectInstance(title) {
+    cy.do(MultiColumnListCell(title).click());
+  },
+
+  toggleAccordionItemsButton(holdingId) {
+    cy.do(Button({ id: `accordion-toggle-button-${holdingId}` }).click());
+  },
+
+  verifyItemWithStatusExists(holdingId, status) {
+    cy.expect(MultiColumnList({ id: `list-items-${holdingId}` }).find(MultiColumnListCell(status)).exists());
```

Comment:
Are you sure that this method actually verifies the status?

Reply:
yep)